uaclient/security.py

```diff
@@ -740,17 +777,36 @@ def _handle_released_package_fixes(
     if src_pocket_pkgs:
         for pocket in [
             UBUNTU_STANDARD_UPDATES_POCKET,
-            UA_APPS_POCKET,
             UA_INFRA_POCKET,
+            UA_APPS_POCKET,
         ]:
             pkg_src_group = src_pocket_pkgs[pocket]
             binary_pkgs = binary_pocket_pkgs[pocket]
+            pkg_status_override = None
 
             if upgrade_status:
+                # we don't want to proceed with services that are still
+                # in beta. The reason is that we are unable to use those
+                # services by default, meaning that the user could potentially
+                # attach to a subscription and not install the fixes.
+                beta_service = False
+                if binary_pkgs and _is_pocket_used_by_beta_service(
```

Comment:
I think we can drop this logic in general if we are doing the beta checks in `merge_usn_released_binary_package_versions`